### PR TITLE
ParkingSlot Index action is added

### PR DESCRIPTION
**Pull request by SardorBozorov**

### CODE REVIEW

File: ParkingZone.Domain/Entities/ParkingSlot.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+namespace Parking_Zone.Domain.Entities;
+
+public class ParkingSlot
+{
+    [Key]
+    [Required]
+    public long Id { get; set; }
+    [Required]
+    public string Number { get; set; }
```
Comment (dilmurod-rozikov) [inline ParkingZone.Domain/Entities/ParkingSlot.cs]: public int Number { get; set; }
uint bolsa zo'r bo'ladi chunki u negative bolishi keremas

File: ParkingZone/Models/ParkingSlotVMs/ListOfSlotsVM.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+using System.ComponentModel.DataAnnotations;
+
+namespace Parking_Zone.MVC.Models.ParkingSlotVMs;
+
+public class ListOfSlotsVM
+{
+    [Required]
+    public long Id { get; set; }
+    [Required]
+    public string Number { get; set; }
```
Comment (dilmurod-rozikov) [inline ParkingZone/Models/ParkingSlotVMs/ListOfSlotsVM.cs]: uint here also

File: Parking_Zone.Test/Controllers/ParkingSlotControllerTest.cs [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    #region Index
+    [Fact]
+    public void GivenParkingZoneId_WhenIndexIsCalled_ThenReturnsParkingSlotsVM()
+    {
+        // Arrange
+        var expectedVMs = new List<ListOfSlotsVM>();
+        expectedVMs.AddRange(ListOfSlotsVM.MapToVM(_parkingSlot));
+
+        _parkingSlotServiceMoq.Setup(x => x.GetSlotsByZoneId(_id)).Returns(_parkingSlot);
+        _parkingZoneServiceMoq.Setup(x => x.GetById(_id)).Returns(_parkingZone);
```
Comment (dilmurod-rozikov) [inline Parking_Zone.Test/Controllers/ParkingSlotControllerTest.cs]: Verify this one also in Assert

**Review by dilmurod-rozikov — CHANGES_REQUESTED** [review]

File: Parking_Zone.Test/Controllers/ParkingSlotControllerTest.cs [hunk 4]
```diff
@@ -59,8 +59,9 @@ public void GivenParkingZoneId_WhenIndexIsCalled_ThenReturnsParkingSlotsVM()
         var expectedVMs = new List<ListOfSlotsVM>();
         expectedVMs.AddRange(ListOfSlotsVM.MapToVM(_parkingSlot));
 
-        _parkingSlotServiceMoq.Setup(x => x.GetSlotsByZoneId(_id)).Returns(_parkingSlot);
-        _parkingZoneServiceMoq.Setup(x => x.GetById(_id)).Returns(_parkingZone);
```
Comment (dilmurod-rozikov) [inline Parking_Zone.Test/Controllers/ParkingSlotControllerTest.cs]: _parkingZoneServiceMoq.Setup(x => x.GetById(_id)).Returns(_parkingZone);
shunga ham pasda verify yozib qoying
        _parkingSlotServiceMoq.Verify(_parkingSlot => _parkingSlot.GetById(_id), Times.Once);
shunaqa

**Review by dilmurod-rozikov — CHANGES_REQUESTED** [review]

File: Parking_Zone.Test/Controllers/ParkingSlotControllerTest.cs (hunk 4 above)
Comment (SardorBozorov) [reply Parking_Zone.Test/Controllers/ParkingSlotControllerTest.cs]: I have not used ParkingZoneService in ParkingSlot  so I deleted it from there

**Review by dilmurod-rozikov — APPROVED** [review]

**Review by abdullokhtolibjonov — APPROVED** [review]

**Pull request merged**